### ISSUE TITLE
13.5.1 support

### ISSUE THREAD

**Issue by PangeaSoftware** [iGhibli/iOS-DeviceSupport#106]
Does anyone have the 13.5.1 files?  It seems 13.5 is the latest uploaded here, but 13.5.1 is what's on my iPhone.

**Comment by IstvanM** [comment]
> Xcode 11.5 (lastest version available on macOS AppStore) supports 13.5.1. No need to add files, just update your Xcode or download 13.5 Support file for 11.4.1 or lower.

My company doesn't allow me to install newer versions yet... I would love to have these files..

**Comment by kdurant35** [comment]
@IstvanM you can download 13.5 Support file from [here](https://github.com/iGhibli/iOS-DeviceSupport/blob/master/DeviceSupport/13.5.zip).

**Comment by PangeaSoftware** [comment]
Yes, 13.5 has been there for a while.  It's 13.5.1 that we're looking for.

**Comment by diesal11** [comment]
Using the 13.5 Support Files in Xcode 11.3.1 on MacOS 10.14.x worked for my iPhone running 13.5.1 👍

**Comment by alpamys-qanybet** [comment]
> Using the 13.5 Support Files in Xcode 11.3.1 on MacOS 10.14.x worked for my iPhone running 13.5.1 👍

where do you put the downloaded zip file content. I've put it in Library->Developer->Xcode->iOS Device Support. But my Xcode does not see it.

**Comment by diesal11** [comment]
I put the extracted 13.5 folder in `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/`

**Comment by kdurant35** [comment]
@PangeaSoftware as a metioned before 13.5 files supports iOS 13.5.1

**Comment by tyoc213** [comment]
> I put the extracted 13.5 folder in `/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/`, then restarted Xcode

So 13.5 should work also for 13.5.1?
